### ISSUE TITLE
Scene-Text-Recognition-SSD的前世今生

### ISSUE THREAD

**Issue by wanghaisheng** [wanghaisheng/awesome-ocr#25]
https://github.com/chongyangtao/Awesome-Scene-Text-Recognition

https://github.com/whitelok/image-text-localization-recognition

**Comment by wanghaisheng** [comment]
[ [置顶] SSD: Signle Shot Detector 用于自然场景文字检测 ](http://blog.csdn.net/u010167269/article/details/52851667)
Detect text in natural images with SSD, Single Shot Detection
https://github.com/chenxinpeng/SSD_scene_text_detection
![](https://github.com/chenxinpeng/SSD_scene-text-detection/raw/master/test_file/output_120.png)

**Comment by wanghaisheng** [comment]
https://github.com/MichalBusta/FASText
http://cmp.felk.cvut.cz/%7Eneumalu1/neumann_iccv2015.pdf

SSD-based plate detection
https://github.com/hyh21521038/ssd-plate_detection

**Comment by wanghaisheng** [comment]
https://github.com/moritzschaefer/unsupervised-text-detection 
Implementation of the paper "Text Detection and Character Recognition in Scene Images with Unsupervised Feature Learning" from Andrew Ng et. al.

**Comment by wanghaisheng** [comment]
https://github.com/SophieFeng/Scene-Text-Label-Handbook 
场景文本标注手册 v1.4.docx

**Comment by wanghaisheng** [comment]
Text detection and recognition from natural scene using stroke width transform and deep feature classification. 

https://github.com/ishtiakzaman/TextRecognitionFromNaturalScene

**Comment by wanghaisheng** [comment]
https://github.com/jugg1024/Text-Detection-with-FRCN

Text-Detection-using-py-faster-rcnn-framework

**Comment by wanghaisheng** [comment]
图像语义分割之特征整合和结构预测  
http://blog.csdn.net/u012759136/article/details/59115476

**Comment by wanghaisheng** [comment]
文本定位
https://github.com/vipul-sharma20/sharingan
http://www.vipul.xyz/2017/03/sharingan-newspaper-text-and-context.html


Text-Detection-using-py-faster-rcnn-framework
https://github.com/jugg1024/Text-Detection-with-FRCN



LBP级联+CNN 回归定位车牌
https://github.com/szad670401/Rubost-Chinese-License-Plate-Locate-Using-LBP-adaboost-with-CNN-regression


EasyPR采用的车牌定位算法。CMER代表文字定位方法，SOBEL和COLOR分别代表边缘和颜色定位方法
https://github.com/liuruoze/EasyPR/issues/31

**Comment by wanghaisheng** [comment]
connectionist text  proposal network for scene text detection
https://arxiv.org/abs/1609.03605
http://www.cnblogs.com/lillylin/p/6277061.html
https://github.com/tianzhi0549/CTPN
文字检测与识别资料整理（数据库，代码，博客）【持续更新】
http://www.cnblogs.com/lillylin/p/6893500.html
Detecting Text in Natural Image with Connectionist Text Proposal Network
We propose a novel Connectionist Text Proposal Network (CTPN) that accurately localizes text lines in natural image. The CTPN detects a text line in a sequence of fine-scale text proposals directly in convolutional feature maps. We develop a vertical anchor mechanism that jointly predicts location and text/non-text score of each fixed-width proposal, considerably improving localization accuracy. The sequential proposals are naturally connected by a recurrent neural network, which is seamlessly incorporated into the convolutional network, resulting in an end-to-end trainable model. This allows the CTPN to explore rich context information of image, making it powerful to detect extremely ambiguous text. The CTPN works reliably on multi-scale and multi- language text without further post-processing, departing from previous bottom-up methods requiring multi-step post-processing. It achieves 0.88 and 0.61 F-measure on the ICDAR 2013 and 2015 benchmarks, surpass- ing recent results [8, 35] by a large margin. The CTPN is computationally efficient with 0:14s/image, by using the very deep VGG16 model [27]. Online demo is available at: this http URL

**Comment by wanghaisheng** [comment]
【PyTorch实现的Single Shot MultiBox Object Detector(SSD)】’A PyTorch Implementation of Single Shot MultiBox Detector.' by Max deGroot GitHub: 
https://github.com/amdegroot/ssd.pytorch

**Comment by wanghaisheng** [comment]
Detecting Text in Natural Image with Connectionist Text Proposal Network
https://link.springer.com/chapter/10.1007/978-3-319-46484-8_4
http://mmlab.siat.ac.cn/yuqiao/

https://github.com/yjxiong/temporal-segment-networks
http://image-net.org/challenges/LSVRC/2015/results#scene
http://lsun.cs.princeton.edu/leaderboard/index_2016.html
https://research.mapillary.com/lsun.html

**Comment by wanghaisheng** [comment]
https://zhuanlan.zhihu.com/p/35642094
SSD 系列论文总结
